next/src/components/ThreeDotLoader.jsx

```diff
@@ -1,11 +1,13 @@
 'use client'
 
-import Loader from 'react-loader-spinner';
```

Comment:
Could we keep this 3 dot loader in the /ui directory? General purpose reusable components should live there and custom made ones for a particular page should be in /components